source/imaer-gml/src/test/java/nl/overheid/aerius/gml/GMLRoundtripTest.java

```diff
@@ -248,6 +247,19 @@ private ImportParcel importAndCompare(final String versionString, final String f
     }
   }
 
+  private static void revertAutoCorrections(final ImportParcel result) {
+    if (result.getSituation().getType() == null) {
+      result.getSituation().setType(SituationType.PROPOSED);
+    }
+  }
+
+  private static String revertAutoCorrections(final String gml, final AeriusGMLVersion targetVersion) {
+    if (targetVersion.ordinal() >= AeriusGMLVersion.V5_1.ordinal()) {
+      return gml.replaceAll(SituationType.OFF_SITE_REDUCTION.name(), LegacySituationType.NETTING.name());
+    }
+    return gml;
+  }
```

Comment:
Is this still needed?

Reply:
unfortunately yes because the test gmls contain `NETTING` but after reading and writing this will now always become `OFF_SITE_REDUCTION` so the roundtrip is not completely the same. I've build in this check on the version assuming that gml version 5.2 and up will never contain `NETTING` as situationtype.

Reply:
Then I wouldn't mind if 5.1 roundtrip GML already contained `OFF_SITE_REDUCTION` instead. That way you don't have to do these auto corrections I think? When reading older versions it should be converted, and when writing that one again it should contain the new value.